commands/server.go

```diff
@@ -32,12 +32,14 @@ import (
 	"github.com/spf13/viper"
 )
 
-var serverPort int
-var serverInterface string
-var serverWatch bool
-var serverAppend bool
-var disableLiveReload bool
-var renderToDisk bool
+var (
+	serverPort        int
+	serverInterface   string
+	serverWatch       bool
+	serverAppend      bool
+	disableLiveReload bool
+	renderToDisk      bool
+)
```

Comment:
Thanks, I actually looked for the "sort function" in my Go ide (liteide), didn't find it ...